ports/ripper37-libbase/portfile.cmake

```
@@ -0,0 +1,32 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO RippeR37/libbase
+    REF "v${VERSION}"
+    SHA512 7a1c77815634b8b07f324bcd6bb4a9b0804edb216bdd3f444265c97efdfc1dadd5fb2cf132987626872170ef6fcff5e16e3e1a79246c6107f91cb3192bf1b679
+    HEAD_REF master
+    PATCHES
+        01_fix_output_lib_name_conflict.patch
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DLIBBASE_OUTPUT_NAME=ripper37-libbase
+        -DLIBBASE_CODE_COVERAGE=OFF
+        -DLIBBASE_BUILD_DOCS=OFF
+        -DLIBBASE_CLANG_TIDY=OFF
+        -DLIBBASE_BUILD_EXAMPLES=OFF
+        -DLIBBASE_BUILD_TESTS=OFF
+        -DLIBBASE_BUILD_PERFORMANCE_TESTS=OFF
+)
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(PACKAGE_NAME "ripper37-libbase")

```

~~~
ripper37-libbase:x64-windows-static:/share/ripper37-libbase/libbase-config.cmake
~~~
AFAIU the intented package name is to remain unchanged `libbase`. So the CMake config must be in a directory name starting with `libbase`.
```suggestion
vcpkg_cmake_config_fixup(PACKAGE_NAME "libbase")
```
I didn't check if you need to update the patch and/or add `CONFIG_PATH <where_from>` here.

---

You port will own two dirs in `share`. It is not so unusual with CMake files.

---

Ok, reverted the changes that tried to merge those, package should now be `libbase` as it was originally. Thank you!

I also tested it locally both with `vcpkg install ripper37-libbase` (on modified vcpkg repo) and via custom-overlay in manifest mode to load and use `libbase` in some example project and I found a bug (in original library) that broke building if gtest/benchmark weren't available (I had a wrong impression about using `find_dependency()` calls in cmake config file, I need to fix it...).

For now I added a patch for this that is fine for vcpkg, I'll fix it in the `libbase` in future version.